src/libraries/Microsoft.PowerFx.Core/Public/Values/PrimitiveValueConversions.cs

```diff
@@ -167,6 +167,7 @@ private static bool TryConvertToDouble(object value, out double result)
 
         internal static ErrorValue OverflowError(IRContext irContext)
         {
+            // !!!TODO move this to interpreter

```

Comment:
>!!! [](http://example.com/codeflow?start=15&length=3)

note we shouldn't check these markers in. 

At least, file a github issue and include the link. 
Then we havea  structured way to search for the github issue in code.

Reply:
Replaced by github issue links.